SizeOf/SizeOf/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
```

Comment:
The files should have the standard license comments. See other files in this repo.